config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+07ijCCJG7wO6WmveKE5ye9CCBXhg0sxx0EprtfmTFINPIA3uwRYg08NejSCTn/7jRerNaYq/Ish/tymylOPKW+BUXapPAhI/xTJaSxzplJHMgx9M52FQ9xi7yC/fcHVwz3fk3Gc7ioqwyxqp1aHppp8grQ1QuxqDvB7EYH7FUdVsW+XW8vXkvko3qQm4/N1Ljt92CbCwz6u5H2oDouRg5wtRJFo4MnuNbpHoKcF4RE8HNE6CZM0rVws3p4KcxteieLBg72iRzBmfq24wTRrlMhHWOZWbNvQGxFQ98nFW/3FFD21gFgp6m2L7F+Tn6MIezXeJRIelZBFrE6FgrNE7JRkIp0Vr0vgvFPE/G7LrqSTa0s87Z6bCl+BMWMmFhhpLDHRcTMK12tcbvOsaxgNUdnZUJaj4KW2md4Hi--YRaLQztbx1M7tgGG--b4A8b64jn68tPS8Nhsi/mQ==
```

Comment:
Just a demo app so I'm not taking the step of hiding secrets in ENV vars for now